src/test/java/LoginTest.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.events.EventFiringWebDriver;
+import pages.CustomerLoginPage;
+import pages.LandingPage;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+public class LoginTest {
+
+    private static WebDriver driver;
+
+    @Before
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver",
+                "src/main/resources/drivers/chromedriver.exe");
```

Comment:
Why this is in 2 lines? Should be in one. 
System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver.exe");